utils/build-script-impl

```diff
@@ -211,7 +211,7 @@ KNOWN_SETTINGS=(
     build-jobs                         ""        "The number of parallel build jobs to use"
     darwin-toolchain-alias             ""        "Swift alias for toolchain"
     android-ndk                        ""        "An absolute path to the NDK that will be used as a libc implementation for Android builds"
-    android-ndk-version                ""        "A version of the NDK to use when building for Android. Currently only 21 or above is supported"
+    android-api-level                  ""        "The Android API level to target when building for Android. Currently only 21 or above is supported"
     android-ndk-toolchain-version      ""        "A version of the toolchain to use when building for Android. Use 4.8 for 32-bit builds, 4.9 for 64-bit builds"
```

Comment:
Happy to send a pull request. Seems like you're working on this full time at the moment, and I'm aware that to some degree you're juggling multiple branches (and doing an awesome job at it!) so I'm just checking I'm not missing something that's planned for the future: in the docs we're saying 4.8 shouldn't work at all, in another (see above) we're saying it's not recommended but possible, and here we're saying use it for 32-bit builds.